From 92d0653f09aa44b24b9306fa9384f13d2c9f1a24 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Fri, 18 Jan 2019 22:05:37 +0100
Subject: [PATCH] efilink: fix build with gcc 4.8

intptr_t is undefined without an include on stdint.h

Fixes:
 - http://autobuild.buildroot.org/results/a0ca37b5ed27af445344e3ac49dc87bb17512c50

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://sourceforge.net/p/gnu-efi/code/merge-requests/4]
---
 lib/lib.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/lib.h b/lib/lib.h
index 37fbb95..7858cd0 100644
--- a/lib/lib.h
+++ b/lib/lib.h
@@ -18,6 +18,7 @@ Revision History
 
 #ifdef __GNUC__
 #pragma GCC visibility push(hidden)
+#include <stdint.h>
 #endif
 
 #include "efi.h"
-- 
2.7.4

